### PR TITLE
[ISSUE #2884] add shenyu-examples-springmvc-tomcat

### PR DESCRIPTION
**Pull request by dragon-zhang**
see https://github.com/apache/incubator-shenyu/issues/2884 and https://github.com/apache/incubator-shenyu/pull/2875

### CODE REVIEW

File: shenyu-examples/shenyu-examples-springmvc-tomcat/pom.xml [hunk 1]
```diff
@@ -0,0 +1,56 @@
...
+    <parent>
+        <artifactId>shenyu-examples</artifactId>
+        <groupId>org.apache.shenyu</groupId>
+        <version>2.4.3-SNAPSHOT</version>
+    </parent>
+    <modelVersion>4.0.0</modelVersion>
+    <artifactId>shenyu-examples-springmvc-tomcat</artifactId>
+    <packaging>war</packaging>
+
+    <properties>
```
Comment (yu199195) [inline shenyu-examples/shenyu-examples-springmvc-tomcat/pom.xml:31]: remove this

File: shenyu-examples/shenyu-examples-springmvc-tomcat/src/main/java/org/apache/shenyu/examples/springmvc/controller/HttpTestController.java [hunk 2]
```diff
@@ -0,0 +1,215 @@
+package org.apache.shenyu.examples.springmvc.controller;
+
+/*
```
Comment (yu199195) [inline shenyu-examples/shenyu-examples-springmvc-tomcat/src/main/java/org/apache/shenyu/examples/springmvc/controller/HttpTestController.java:3]: move this license to top

**Comment by dragon-zhang** [comment]
Got it.

File: shenyu-examples/shenyu-examples-springmvc-tomcat/src/main/java/org/apache/shenyu/examples/springmvc/controller/HttpTestController.java (hunk 2 above)
Comment (dragon-zhang) [reply shenyu-examples/shenyu-examples-springmvc-tomcat/src/main/java/org/apache/shenyu/examples/springmvc/controller/HttpTestController.java:1]: done

File: shenyu-examples/shenyu-examples-springmvc-tomcat/pom.xml (hunk 1 above)
Comment (dragon-zhang) [reply shenyu-examples/shenyu-examples-springmvc-tomcat/pom.xml:30]: done

Comment (yu199195) [reply shenyu-examples/shenyu-examples-springmvc-tomcat/pom.xml:30]: not commits？

**Review by yu199195 — APPROVED** [review]

**Pull request merged**